src/lib/routeParams.ts

```diff
@@ -147,12 +150,14 @@ export const useRouteParams = () => {
         folderNames.push(name)
       }
 
+      const lastItem = restNames.pop()
       return {
         name: 'storages.notes',
         storageId,
         folderPathname:
           folderNames.length === 0 ? '/' : '/' + folderNames.join('/'),
         noteId,
+        editorLineFocus: parseInt(lastItem ? lastItem : '0'),
```

Comment:
Should do the nully check more strictly. `lastItem != null`

Reply:
Done, i.e. removed editorLineFocus fully, since hash from router is now used.